<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Handwriting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
    <item>
     <widget class="CanvasWidget" name="canvasWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>320</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="outputLayout">
      <item>
       <layout class="QVBoxLayout" name="rawLayout">
        <item>
         <layout class="QHBoxLayout" name="rawHeader" stretch="1,1,3">
          <item>
           <widget class="QLabel" name="rawLabel">
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="rawComboBox"/>
          </item>
          <item>
           <spacer name="rawSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="rawText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="rawPinyin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="translatedLayout" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="translatedHeader" stretch="1,1,3">
          <item>
           <widget class="QLabel" name="translatedLabel">
            <property name="text">
             <string>Translated</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="translatedComboBox"/>
          </item>
          <item>
           <spacer name="translatedSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="translatedText">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_clear"/>
    <addaction name="action_reset"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuBrush">
    <property name="title">
     <string>Brush</string>
    </property>
    <widget class="QMenu" name="menuBrushSize">
     <property name="title">
      <string>Size</string>
     </property>
     <addaction name="action_3px"/>
     <addaction name="action_5px"/>
     <addaction name="action_7px"/>
     <addaction name="action_9px"/>
    </widget>
    <addaction name="menuBrushSize"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_tablet"/>
    <addaction name="action_fullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBrush"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="action_clear">
   <property name="text">
    <string>Clear Canvas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action5px">
   <property name="text">
    <string>5px</string>
   </property>
  </action>
  <action name="action7px">
   <property name="text">
    <string>7px</string>
   </property>
  </action>
  <action name="action9px">
   <property name="text">
    <string>9px</string>
   </property>
  </action>
  <action name="action_3px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3px</string>
   </property>
  </action>
  <action name="action_5px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5px</string>
   </property>
  </action>
  <action name="action_7px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7px</string>
   </property>
  </action>
  <action name="action_9px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9px</string>
   </property>
  </action>
  <action name="action_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>en</string>
   </property>
  </action>
  <action name="action_ja">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ja</string>
   </property>
  </action>
  <action name="action_ko">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ko</string>
   </property>
  </action>
  <action name="action_zh_cn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh_CN</string>
   </property>
  </action>
  <action name="action_zh_TW">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh_TW</string>
   </property>
  </action>
  <action name="action_tablet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tablet Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>canvas_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
